content/en/ebpf.md

```diff
@@ -0,0 +1,20 @@
+---
+title: eBPF
+status: Completed
+category: architecture
+---
+
+## What it is
+
```

Comment:
```suggestion
eBPF, or extended Berkeley Packet Filter, is a technology that allows applications to run within the kernel of a Linux system without changing the kernel source code or load Linux kernel modules.
```
I think the first sentence must clearly state what it is. We can then break that down and explain.